src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -53,18 +53,7 @@
         "doctrine/collections": "<1.8",
         "doctrine/dbal": "<3.6",
         "doctrine/lexer": "<1.1",
-        "doctrine/orm": "<2.15",
-        "symfony/cache": "<6.4",
-        "symfony/dependency-injection": "<6.4",
-        "symfony/form": "<6.4.6|>=7,<7.0.6",
-        "symfony/http-foundation": "<6.4",
-        "symfony/http-kernel": "<6.4",
-        "symfony/lock": "<6.4",
-        "symfony/messenger": "<6.4",
-        "symfony/property-info": "<6.4",
-        "symfony/security-bundle": "<6.4",
-        "symfony/security-core": "<6.4",
-        "symfony/validator": "<6.4"
```

Comment:
conflicts are terrible
it's not because we don't test with older versions that they don't work
this leads to real issues like https://github.com/symfony/symfony/pull/58997
so we'd rather not use them

Reply:
We have a CI that run tests cross-versions instead, so that even if not allowed by require-dev, we do test outside these boundaries (as hinted in https://github.com/symfony/symfony/pull/58997 again)